### DISCUSSION TITLE
Fluree and RDF star

### DISCUSSION THREAD

**Discussion by Tr3yb0** [fluree/db#100] [Q&A]
How is Fluree precisely related to RDF star?

**Comment by Tr3yb0** [comment]
Fluree does not support RDF star, but we have plans to. 
RDF Star is designed to hold metadata, and we already store metadata around RDF - through the time/provenance activities as well as the generic 'meta' metadata part attached to RDF.

**Comment by djradon** [comment]
Any update on RDF-star support?

**Reply by bplatz** [reply]
Hi @djradon - no support yet or planned, but we have some internals designed to add support when/if we move in that direction.

One of the most common use cases for RDF star is to store data provenance or transactional metadata - but these are already supported in Fluree at a more foundational level, so we wouldn't use RDF star for that use case at all.
